include/random_walks/exponential_hamiltonian_monte_carlo_walk.hpp

```diff
@@ -0,0 +1,205 @@
+// VolEsti (volume computation and sampling library)
+// Copyright (c) 2021 Vissarion Fisikopoulos
+// Copyright (c) 2021 Apostolos Chalkis
+
+
+// Licensed under GNU LGPL.3, see LICENCE file
+
+#ifndef RANDOM_WALKS_EXPONENTIAL_HMC_WALK_HPP
+#define RANDOM_WALKS_EXPONENTIAL_HMC_WALK_HPP
+
+#include "sampling/sphere.hpp"
+
+
+// Exact HMC for sampling from the Exponential distribution restricted to a convex polytope
+
+struct HMCExponentialWalk
```

Comment:
I think here you should change the name of the struct to agree with the one that's already in the develop branch, namely HMCExponentialWalk -> HamiltonianMonteCarloExactExponentialWalk.

Reply:
Thanks for the comment. I renamed it to "ExponentialHamiltonianMonteCarloExactWalk".

Reply:
then please update the name of the file too.